extensions/amp-story-shopping/validator-amp-story-shopping.protoascii

```diff
@@ -19,24 +19,52 @@ html_format: AMP
   amp_layout: {
     supported_layouts: CONTAINER
   }
+  attrs: {
+    name: "data-product-id"
+    mandatory: true
+  }
 }
-tags: {  # <amp-story-shopping-config>
+tags: {  # <amp-story-shopping-attachment>
   html_format: AMP
-  tag_name: "AMP-STORY-SHOPPING-CONFIG"
+  tag_name: "AMP-STORY-SHOPPING-ATTACHMENT"
   requires_extension: "amp-story-shopping"
   attr_lists: "extended-amp-global"
   spec_url: "https://amp.dev/documentation/components/amp-story-shopping/"
+  attrs: {
+    name: "src"
+    value_url: {
+      protocol: "https"
+      allow_relative: true
+      allow_empty: true
+    }
+  }
+  attrs: { name: "theme" }
+  child_tags: {
+    child_tag_name_oneof: "SCRIPT"
+    mandatory_min_num_child_tags: 1
+  }
   amp_layout: {
-    supported_layouts: NODISPLAY
+    supported_layouts: CONTAINER
   }
 }
-tags: {  # <amp-story-shopping-attachment>
+tags: {  # <amp-story-shopping-attachment> (json config)
```

Comment:
It looks like the naming convention for script children is the component name without `<>`. I'm looking at amp-auto-ads as an example.
https://github.com/ampproject/amphtml/blob/main/extensions/amp-auto-ads/validator-amp-auto-ads.protoascii#L1